RecoLocalFastTime/FTLRecProducers/plugins/MTDTrackingRecHitProducer.cc

```diff
@@ -67,7 +68,7 @@ void MTDTrackingRecHitProducer::produce(edm::Event& evt, const edm::EventSetup&
   geom_ = geom.product();
 
   edm::ESHandle<MTDClusterParameterEstimator> cpe;
-  es.get<MTDCPERecord>().get("MTDCPEBase", cpe);
+  es.get<MTDCPERecord>().get("MTDCPEFromSiPMTime", cpe);
```

Comment:
also confused about this change

Reply:
Hi,
Just like the previous problem, the only difference between the "MTDCPEBase" and "MTDCPEFromSiPMTime" is just that "MTDCPEFromSiPMTime" added a more detailed position error information for MTD BTL. I think it's not more specific but just included position error more precisely. The reason I didn't change the "MTDCPEBase" is that I am not sure whether there are some other modules that are also using this class so changing it directly might cause some problems.

Thanks.

Reply:
Does "SiPMTime" make sense for ETL? If it's meaningful only for BTL, perhaps a better name is needed

Reply:
Hi,
"SiPMTime" doesn't make sense for ETL. So maybe I should change the name of that class.

Also, I have a question about this. What is the difference between TrackingRecHits and normal RecHits? Is it fine if TrackingRecHits in BTL have the same positionError with normal RecHits in BTL?

Thanks.

Reply:
please clarify what you mean by "normal RecHit". There is a C++ type ```TrackingRecHit```.

Reply:
Hi,
Sorry for not being clear, I meant `FTLRecHit` for "normal RecHit" and `MTDTrackingRecHit` for "TrackingRecHit".
Thanks!

Reply:
> Also, I have a question about this. What is the difference between TrackingRecHits and normal RecHits? Is it fine if TrackingRecHits in BTL have the same positionError with normal RecHits in BTL?

The FTLRecHits are more like individual "channel" rec hits.
TrackingRecHits are used for tracking (they will need CPE), they are a somewhat generalized representation of FTLCluster with relevant details for trajectory fitting.

Reply:
Cool. I think it should then make sense to make TrackingRecHits have the same positionError with FTLRecHits.
Thanks!